### ISSUE TITLE
docs: how stars works

### ISSUE THREAD

**Issue by davidhodge931** [davidhodge931/ggblanket#333]
```
stars::read_stars(system.file("tif/L7_ETMs.tif", package = "stars")) |>
  sf::st_as_sf(long = TRUE) |>
  gg_sf(col = L7_ETMs.tif,
        facet = band,
        titles = \(x) x)
```

**Comment by davidhodge931** [comment]
```
stars::read_stars(system.file("tif/L7_ETMs.tif", package = "stars")) |>
  terra::rast() %>% 
  ggplot2::fortify()	%>%
  pivot_longer(contains("band")) %>% 
  gg_raster(x = x, 
            y = y, 
            col = value, 
            facet = name, 
            void = TRUE, 
            facet_labels = stringr::str_to_sentence)
```

**Comment by davidhodge931** [comment]
```
stars::read_stars(system.file("tif/L7_ETMs.tif", package = "stars")) |>
  tibble::as_tibble() %>% 
  gg_raster(x = x, 
            y = y, 
            col = L7_ETMs.tif, 
            facet = band, 
            void = TRUE, 
            titles = \(x) x)
```

**Comment by davidhodge931** [comment]
Fix #334 first

**Comment by davidhodge931** [comment]
Added example to gg_raster
